wpimath/src/main/native/include/frc/controller/ElevatorFeedforward.h

```diff
@@ -113,6 +126,106 @@ class ElevatorFeedforward {
            units::volt_t{feedforward.Calculate(r, nextR)(0)};
   }
 
+  /**
+   * Calculates the feedforward from the gains and setpoint assuming discrete
+   * control. Use this method when the setpoint does not change.
+   *
+   * @param setpoint The velocity setpoint, in distance per
+   *                        second.
+   * @return The computed feedforward, in volts.
+   */
+  constexpr units::volt_t Calculate(units::unit_t<Velocity> setpoint) const {
+    return Calculate(setpoint, setpoint);
+  }
+
+  /**
+   * Calculates the feedforward from the gains and setpoints assuming discrete
+   * control.
+   *
+   * <p>Note this method is inaccurate when the velocity crosses 0.
+   *
+   * @param currentVelocity The current velocity setpoint, in distance per
+   *                        second.
+   * @param nextVelocity    The next velocity setpoint, in distance per second.
+   * @return The computed feedforward, in volts.
```

Comment:
We exclude units from parameter docs in C++ where the typename itself documents it.